src/integration/pages/onDemandRadioEpisode/ampTests.js

```diff
@@ -3,4 +3,11 @@ import { runCoreAmpTests, runAmpAnalyticsTests } from '../../common';
 export default () => {
   runCoreAmpTests();
   runAmpAnalyticsTests();
+  it('I can see the hero image on amp', () => {
+    const imageEl = document.querySelector('main amp-img');
```

Comment:
this can be in cross platform if you do `const imageEl = document.querySelector('main amp-img, main img')`

Reply:
Nice one - thanks Jonathan. Added to the common tests https://github.com/bbc/simorgh/pull/6610/files#diff-6b8438eed335be82e997f3a0f261b1eeR26-R32